subscription_manager.go

```diff
@@ -757,7 +751,7 @@ func createPNFilesEvent(filePayload interface{}, m *SubscriptionManager, actualC
 		Publisher:         issuingClientID,
 		UserMetadata:      userMetadata,
 		SpaceId:           spaceId,
-		MessageType:       messageType,
+		Type:              typ,
```

Comment:
same here?
```suggestion
		Type:              type,
```

Reply:
As stated in previous comment. Not typo. Keyword. That's just life :D

Reply:
Wait. This actually look like a mistake